types/default-theme.d.ts

```diff
@@ -81,6 +56,48 @@ export namespace DefaultTheme {
      * The carbon ads options. Leave it undefined to disable the ads feature.
      */
     carbonAds?: CarbonAdsOptions
+
+    /**
+     * Custom labels
+     */
+    translations?: {
```

Comment:
Maybe we should name this more like "Labels"? 🤔 Because customizing these labels doesn't necessarily mean those are translated. For example, changing `outlineTitle` to "Table of Contents".